src/ifupdown2/Makefile

```diff
@@ -0,0 +1,29 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = $(IFUPDOWN2)
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Remove any stale files
+	rm -rf ./ifupdown2-$(IFUPDOWN2_VERSION) ./$(IFUPDOWN2_VERSION).tar.gz
+
+	# Get ifupdown2 release
+	wget --no-check-certificate https://github.com/CumulusNetworks/ifupdown2/archive/$(IFUPDOWN2_VERSION).tar.gz
+	tar -z -f $(IFUPDOWN2_VERSION).tar.gz -x
+	pushd ./ifupdown2-$(IFUPDOWN2_VERSION)
+
+	git init
+	git add -f *
+	git commit -m "unmodified ifupdown2 source"
```

Comment:
we don't need any of git commands unless you're going to use stg.

Reply:
Agreed. Will remove these.